src/main/java/com/github/huangp/entityunit/entity/EntityMakerImpl.java

```diff
@@ -129,7 +135,13 @@ private static void putManySideEntityIfExists(Object entity, Settable element, B
         Optional keyOptional = holder.tryGet(keyType);
         Optional valueOptional = holder.tryGet(valueType);
         if (!keyOptional.isPresent()) {
-            log.warn("You have to manually resolve this: {} {}", element.getType(), element);
+            String resolvethis = String.format("You have to manually resolve this: {} {}",
```

Comment:
The '{}' in string template is only understandable by log. String.format uses '%s'. Also String.format will always call toString on the object event if log has been disabled. I'd say just duplicate the message in the log.warn and log.info.